tests/zfs-tests/tests/functional/l2arc_arcstats/l2arc_arcstats_001_pos.ksh

```diff
@@ -0,0 +1,99 @@
+#!/bin/ksh -p
```

Comment:
we've been trying to avoid the number in the test filename. It is really meaningless and we're better served using a more descriptive filename. IMHO, l2arc_arcstats_pos.ksh is sufficient

Reply:
Will fix.